src/DataTable/DataTable.jsx

```diff
@@ -14,38 +14,46 @@ export class DataTable extends Component {
   }
   
   static defaultProps = {
-    autoResize: false
+    autoResize: true
   }
-  
-  handleResize (e) {
+
+  handleResize (grid) {
```

Comment:
I don't want to be too picky, and I missed this in the last code review, but I would write this like this. I realize, that it is technically longer this way, however, its easier to add to this in the future if we need to for some reason. 

```javascript
handleResize (grid) {
  if (!this.autoResize) {
    return
  }

  grid.api.sizeColumnsToFit()
}
```